ocaml/libs/clock/date.ml

```diff
@@ -60,30 +56,39 @@ let best_effort_iso8601_to_rfc3339 x =
   in
   match tz with
   | None | Some "" ->
-      (* the caller didn't specify a tz. we must try to add one so that ptime can at least attempt to parse *)
-      (Printf.sprintf "%sZ" x, Empty)
-  | Some tz ->
-      (x, TZ tz)
+      (* the caller didn't specify a tz, assume Universal Local Time *)
```

Comment:
Is Universal Local Time a fixed concept? It sounds contradictory.

Reply:
It's not, it should read Universal Time Zone

Reply:
Now it reads Coordinated Universal Time